tests/mongo-go-driver.yml

```diff
@@ -132,3 +132,4 @@ results:
       - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/RunCommandExamples
       - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/StableAPExamples
       - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/UpdateExamples
+      
```

Comment:
```suggestion
```

Reply:
So we are removing the newline now?

Reply:
The last line of the file should be not empty and end with \n

Reply:
<img width="77" alt="CleanShot 2022-11-25 at 14 59 46@2x" src="https://user-images.githubusercontent.com/11512/203969883-87ea578a-7412-4f21-b61c-df09727fd5cb.png">

That means that line does not end with \n